include/envoy/common/platform.h

```diff
@@ -235,3 +235,15 @@ struct mmsghdr {
   unsigned int msg_len;
 };
 #endif
+
+#ifdef __linux__
+#define SUPPORTS_PTHREAD_GETNAME_NP
```

Comment:
suggest defining it to 1 or 0, rather than defining/not-defining it. That would make it less cumbersome to use in logical expressions at call-sites.

I think that's more consistent with other similar definitions in this file.